Classes/Command/ProjectionCommandController.php

```diff
@@ -116,11 +116,16 @@ public function replayCommand($projection, $quiet = false)
             $this->output->progressStart();
         }
         $eventsCount = 0;
-        $this->projectionManager->replay($projectionDto->getIdentifier(), function () use (&$eventsCount, $quiet) {
+        $startOfLastThousandEvents = microtime(true);
+        $this->projectionManager->replay($projectionDto->getIdentifier(), function () use (&$eventsCount, $quiet, &$startOfLastThousandEvents) {
             $eventsCount ++;
             if (!$quiet) {
                 $this->output->progressAdvance();
             }
+            if ($eventsCount % 1000 === 0) {
+                $this->outputLine('Time taken for 1000 events: ' . (microtime(true) - $startOfLastThousandEvents));
+                $startOfLastThousandEvents = microtime(true);
```

Comment:
I'm not sure if a configuration option is the best solution: on the console you might want it, but the background worker (i.e. cronjob) doesn't. Let's make it a flag on the command I'd say